src/DynamoCoreWpf/Views/Preview/WatchTree.xaml

```diff
@@ -158,7 +160,45 @@
                                     <Setter Property="Visibility"
                                             TargetName="Expander"
                                             Value="Collapsed" />
+                                    <Setter Property="Margin"
```

Comment:
Trigger for such situation:
![image](https://cloud.githubusercontent.com/assets/8158404/11561141/fc9177b0-99cd-11e5-9cae-db928f3707ac.png)